sdk/cognitivelanguage/azure-ai-language-conversations/azure/ai/language/conversations/models/_models_py3.py

```diff
@@ -384,36 +694,95 @@ class ConversationIntent(msrest.serialization.Model):
 
     :ivar category: Required. A predicted class.
     :vartype category: str
-    :ivar confidence_score: Required. The confidence score of the class from 0.0 to 1.0.
-    :vartype confidence_score: float
+    :ivar confidence: Required. The confidence score of the class from 0.0 to 1.0.
+    :vartype confidence: float
     """
 
     _validation = {
         'category': {'required': True},
-        'confidence_score': {'required': True, 'maximum': 1, 'minimum': 0},
+        'confidence': {'required': True, 'maximum': 1, 'minimum': 0},
     }
 
     _attribute_map = {
         'category': {'key': 'category', 'type': 'str'},
-        'confidence_score': {'key': 'confidenceScore', 'type': 'float'},
+        'confidence': {'key': 'confidenceScore', 'type': 'float'},
     }
 
     def __init__(
         self,
         *,
         category: str,
-        confidence_score: float,
+        confidence: float,
         **kwargs
     ):
         """
         :keyword category: Required. A predicted class.
         :paramtype category: str
-        :keyword confidence_score: Required. The confidence score of the class from 0.0 to 1.0.
-        :paramtype confidence_score: float
+        :keyword confidence: Required. The confidence score of the class from 0.0 to 1.0.
+        :paramtype confidence: float
         """
         super(ConversationIntent, self).__init__(**kwargs)
         self.category = category
-        self.confidence_score = confidence_score
+        self.confidence = confidence
+
+
+class ConversationItemBase(msrest.serialization.Model):
```

Comment:
I don't think this should be called ConversationItem. We generally use the suffix `Item` for listed results.

Reply:
what do you think we should call it then?